extensions/amazon-aws/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Amazon AWS Changelog
 
+## [Console command] - 2024-06-14
+
+- Adds shortcut login for SSO profiles
+
 ## [SQS, CFN, DDB, CW Logs commands] - 2024-06-14
```

Comment:
Two change log entries with the same date. Maybe use this:

```md
## [Console command] - {PR_MERGE_DATE}
```

and @raycastbot will auto update the date in case this gets merged tomorrow. Or you can merge the points to the previous changeling entry.